src/features/users/components/UserContainer/UserContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { userApi } from "../../../../app/services/UserService"
+import UserItem from "../UserItem/UserItem"
+
+
+const UserContainer = () => {
+    const {data: users, error, isLoading} = userApi.useFetchAllUsersQuery(5)
```

Comment:
you shouldn't fetch any APIs -- use selectors instead


get users by selectors added to selectors file